Examples/Tests/darwin_solver_EM_modes/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Add tests (alphabetical order) ##############################################
+#
+
+add_warpx_test(
+    test_1d_darwin_solver_em_modes_picmi  # name
+    1  # dims
+    1  # nprocs
```

Comment:
There is some remaining bug in the implementation - if I run with more than 1 processor (or more than 1 tile) the MS solver does not converge.